index.js

```diff
@@ -1,153 +1,157 @@
-"use strict"
+'use strict';
 
 module.exports = {
-    "env": {
-        "browser": true,
-        "node": false,
-        "es6": true
+    env: {
+        browser: true,
+        node: true,
+        es2022: true,
+        amd: true,
     },
-    "globals": {
-        "require": true,
-        "module": true
+    extends: 'eslint:recommended',
+    plugins: ['import'],
+    parserOptions: {
+        sourceType: 'module'
     },
-    "extends": "eslint:recommended",
-    "parserOptions": {
-        "ecmaVersion": 6,
-        "sourceType": "module"
-    },
-    "rules": {
-        "indent": [
-            "error",
+    rules: {
+        'arrow-parens': ['error', 'always'],
+        'indent': [
+            'error',
             4,
             {
-                "SwitchCase": 1
+                SwitchCase: 1,
             }
         ],
-        "linebreak-style": [
-            "error",
-            "unix"
-        ],
-        "quotes": [
-            "error",
-            "single"
-        ],
-        "one-var": [
-            "error",
-            "never"
-        ],
-        "operator-linebreak": [
-            "error",
-            "before"
-        ],
-        "padded-blocks": [
-            "error",
-            "never"
-        ],
-        "semi": [
-            "error",
-            "always"
-        ],
-        "max-len": [
-            "error",
+        'quotes': ['error', 'single'],
+        'one-var': ['error', 'never'],
+        'operator-linebreak': ['error', 'before'],
```

Comment:
Is this compatible with prettier? But I think it's not a problem.

Reply:
Let's discuss this in the `space-before-function-paren` thread.